hbase-client/src/main/java/org/apache/hadoop/hbase/client/TableDescriptorBuilder.java

```diff
@@ -1600,6 +1600,10 @@ public void removeCoprocessor(String className) {
       // if we found a match, remove it
       if (match != null) {
         ModifyableTableDescriptor.this.removeValue(match);
+      } else {
+        throw new IllegalArgumentException(String
```

Comment:
For branch-2, what if you were to replace this `throw` with a `LOG.warn()` message containing the same text? Would that be sufficient?